### DISCUSSION TITLE
因延迟被扣款多次，请求退掉多付的钱

### DISCUSSION THREAD

**Discussion by 7www7** [getlantern/lantern-forum-cn#896] [General]
我只打算买一年，但因为延迟导致付款了多次，请求退掉多付的钱，9月16日-17日扣的。
账号：939973895@qq.com，银联在线支付的，应该是付了6次，退5次。

**Comment by 7www7** [comment]
![O%K7`2A%M)5Y) NC8CQQ4{A](https://github.com/user-attachments/assets/62966727-6479-4420-a6f3-8d7655a0d197)
![U6MK0$W GC$NG8L34A Y0(B](https://github.com/user-attachments/assets/1f40a4fc-de65-4c72-a808-b5260ea388a8)

**Comment by MaxMKMax** [comment]
@7www7 哇. 您重复支付这么多次. 好的, 请您放心, 我已经帮您提交去处理退款了. 会留着 1 年给您.

**Comment by greySkiesMarsh** [comment]
@7www7 Hi there, refunds for your duplicate purchases have been processed and should be reflected in your account within 5-10 business days.
